scripts/go_gen_p4_const.go

```diff
@@ -0,0 +1,162 @@
+package main
```

Comment:
This should replace the python script. It still needs some tweaking (e.g. string formatting)